src/Uno.UI/UI/Xaml/Controls/Layouter/Layouter.cs

```diff
@@ -735,10 +735,11 @@ protected Thickness MarginChild(View view)
 				childSize += childMarginSize;
 			}
 
-			return childSize
-				.Min(frameSize) // at most
-				.Min(max) // at most
-				.Max(min); // at least
```

Comment:
It looks like these extension methods involved some boxing. I used `Math.Max` and `Math.Min` instead.